Example/GiniVision.xcodeproj/project.pbxproj

```diff
@@ -360,6 +360,7 @@
 						CreatedOnToolsVersion = 6.3.1;
 						DevelopmentTeam = JA825X8F7Z;
 						LastSwiftMigration = 0810;
+						ProvisioningStyle = Automatic;
```

Comment:
No, automatic is fine, but I don't like seeing unrelated changes in a commit. This PR is about Tablet support and changing the provisioning settings is unrelated.